src/app/component/footer/footer.component.scss

```diff
@@ -1,27 +1,23 @@
-.footer{
-    position:absolute;
-    width: 100%;
-    height: 65px;
-    text-align: center;
-    bottom: 0;
+.footer {
+  padding: 5px;
+  position: fixed;
+  background-color: white;
+  width: 100%;
+  text-align: center;
+  bottom: 0px;
 }
-b{
-    color: darkblue;
-}
-a{
-    color: blue;
+
+.footer-link {
+  color: #0000ee;
 }
-@media (max-width: 1300px) {
-    .footer {
-        font-size: 10px !important;
-    }
-    .footer .credits {
-        font-size: 10px !important;
-    }
+.fa {
```

Comment:
why overriding whole fa ?

Reply:
yes I have change 'fa'  name to 'icon' name in html and CSS files